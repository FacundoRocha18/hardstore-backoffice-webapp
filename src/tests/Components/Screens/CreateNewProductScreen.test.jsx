import { CreateNewProduct } from '../../../Components';
import { render, screen } from '@testing-library/react';

describe('Tests on <CreateNewProduct /> component', () => {

	test('should match the snapshot', () => {

		const { container } = render(
			<CreateNewProduct />
		)

		expect(container).toMatchSnapshot();
	})
})