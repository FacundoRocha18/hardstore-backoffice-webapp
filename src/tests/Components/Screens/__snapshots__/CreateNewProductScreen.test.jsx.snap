// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on <CreateNewProduct /> component should match the snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="title"
    >
      <h2>
        Subir nuevo producto
      </h2>
    </div>
    <div
      class="form_container"
    >
      <form>
        <div
          class="form_group"
        >
          <label
            for="product_name"
          >
            Nombre del producto: 
          </label>
          <input
            autocomplete="off"
            id="product_name"
            name="product_name"
            placeholder="Nombre"
            required=""
            type="text"
          />
        </div>
        <div
          class="form_group"
        >
          <label
            for="product_sku"
          >
            Código SKU del producto: 
          </label>
          <input
            autocomplete="off"
            id="product_sku"
            name="product_sku"
            placeholder="SKU"
            required=""
            type="text"
          />
        </div>
        <div
          class="form_group"
        >
          <label
            for="product_qty"
          >
            Cantidad:
          </label>
          <input
            id="product_qty"
            min="1"
            name="product_qty"
            required=""
            step="1"
            type="number"
            value="1"
          />
        </div>
        <div
          class="form_group"
        >
          <label>
            Categoría del producto:
          </label>
          <div
            class="container"
          >
            <div
              class="column"
            >
              <input
                class="selected_cat"
                id="selected_cat"
                name="selected_cat"
                placeholder="Seleccione una categoría"
                required=""
                type="text"
                value=""
              />
              <div
                class="cats_container hidden"
              >
                <ul />
              </div>
            </div>
            <div
              class="button_container"
            >
              <button
                class="deploy_btn"
              >
                <span
                  class="material-icons-round"
                >
                  expand_more
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          class="form_group"
        >
          <label
            for="product_description"
          >
            Descripción del producto: 
          </label>
          <textarea
            cols="80"
            id="product_description"
            name="product_description"
            required=""
            rows="10"
          />
        </div>
        <div
          class="form_group"
        >
          <label
            for="product_image"
          >
            Imagen(es) del producto: 
          </label>
          <div
            class="preview_container"
          >
            <div
              class="images_container"
            >
              <div
                class="image_card no_background"
              >
                <label
                  class="image_preview_label"
                  for="product_image"
                >
                  <span
                    class="material-icons-round"
                  >
                    add
                  </span>
                </label>
              </div>
            </div>
            <input
              accept="image/*"
              class="hidden"
              id="product_image"
              name="product_image"
              required=""
              type="file"
            />
          </div>
        </div>
        <div
          class="price_container"
        >
          <div
            class="form_group"
          >
            <label
              for="product_cost"
            >
              Precio sin IVA:
            </label>
            <input
              id="product_cost"
              min="0"
              name="product_cost"
              required=""
              step="10"
              type="number"
              value="0"
            />
          </div>
          <div
            class="form_group"
          >
            <label
              for="product_iva"
            >
              IVA (25%):
            </label>
            <input
              disabled=""
              id="product_iva"
              name="product_iva"
              type="number"
              value="0"
            />
          </div>
          <div
            class="form_group"
          >
            <label
              for="product_price"
            >
              Precio total (IVA inc.):
            </label>
            <input
              disabled=""
              id="product_price"
              name="product_price"
              required=""
              type="number"
              value="0"
            />
          </div>
        </div>
        <div
          class="buttons_container"
        >
          <button
            class="button save-btn"
          >
            <p>
              Guardar
            </p>
          </button>
          <button
            class="button reset-btn"
          >
            <p>
              Limpiar
            </p>
          </button>
          <button
            class="button cancel-btn"
          >
            <p>
              Cancelar
            </p>
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
